crates/ruff_linter/src/codes.rs

```diff
@@ -1125,8 +1125,10 @@ pub fn code_to_rule(linter: Linter, code: &str) -> Option<(RuleGroup, Rule)> {
         // flake8-logging
         (Flake8Logging, "001") => (RuleGroup::Stable, rules::flake8_logging::rules::DirectLoggerInstantiation),
         (Flake8Logging, "002") => (RuleGroup::Stable, rules::flake8_logging::rules::InvalidGetLoggerArgument),
+        (Flake8Logging, "004") => (RuleGroup::Preview, rules::flake8_logging::rules::ExceptionCallOutsideHandlers),
```

Comment:
Nit: How about `LogExceptionOutsideExceptHandler` and `ExcInfoOutsideExceptHandler`